custom_components/octopus_intelligent/__init__.py

```diff
@@ -60,9 +60,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
 
     #hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, lambda event: octopus_system.stop())
 
-    for component in PLATFORMS:
-        hass.async_create_task(
-            hass.config_entries.async_forward_entry_setup(entry, component))
+    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
```

Comment:
This is not a fix for anything, just an incidental simplification as `async_forward_entry_setups()` includes the `for` loop: https://github.com/home-assistant/core/blob/2024.2.5/homeassistant/config_entries.py#L1566